### PR TITLE
refine brelu; add hardsigmoid, hardswish, maxout; test=develop

### PR DESCRIPTION
**Pull request by zhupengyang**
- 移除2.0中的brelu文档
- hard_sigmoid -> hardsigmoid
- hardswish -> hardswish
- add maxout

image
image
image
image
image
image

### CODE REVIEW

File: doc/paddle/api/paddle/nn/functional/activation/hardshrink_cn.rst [hunk 1]
```diff
@@ -32,6 +32,8 @@ hardshrink激活层。计算公式如下：
 代码示例
 ::::::::::
 
+.. code-block:: python
+
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/nn/functional/activation/hardshrink_cn.rst:36]: 39行不需要了哈

Comment (zhupengyang) [reply doc/paddle/api/paddle/nn/functional/activation/hardshrink_cn.rst:36]: Done

Comment (zhupengyang) [reply doc/paddle/api/paddle/nn/functional/activation/hardshrink_cn.rst:36]: Done

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by hong19860320 — APPROVED** [review]
LGTM

**Pull request merged**